src/pages/GivingFunds/GivingFundsManagementPage.vue

```diff
@@ -0,0 +1,135 @@
+<!--  eslint-disable max-len -->
+<template>
+	<WwwPage main-class="tw-bg-secondary tw-pb-3">
+		<KvPageContainer>
+			<KvGrid class="tw-grid-cols-12 tw--mx-2.5 md:tw-mx-0">
+				<div class="tw-col-span-12 tw-pt-1.5 md:tw-pt-3">
+					<div class="tw-py-2 md:tw-py-3">
+						<h1 class="tw-mb-2 tw-break-words">
+							<div>
+								My Giving Funds
+							</div>
+						</h1>
+					</div>
+					<div
+						v-if="activeFundraisers?.length"
+						class="tw-pb-2 md:tw-pb-3"
+					>
+						<h3 class="tw-mb-2">
+							Active Fundraisers
+						</h3>
+						<div
+							class="tw-rounded tw-p-2 tw-bg-primary"
+						>
+							<KvLoadingPlaceholder
+								v-if="loading"
+								class="md:!tw-mt-1 !tw-w-1/4"
+								style="height: 32px; margin-top: 2px;"
+							/>
+							<div
+								v-for="(fund, i) in activeFundraisers"
+								:key="`active-fundraiser-${i}`"
+							>
+								You hava active fundraiser for <b>{{ fund.name }}</b> <a :href="`/gf-beta/${fund.id}`" target="_blank">View</a>
```

Comment:
`have`

Reply:
Also tracking on link?

Reply:
We're going to rebuild all of the content within this section. This is just to stub out data.